Sources/ConcordiumSwiftSDK/Client.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+import GRPC
+import NIOPosix
+import NIOCore
+
+typealias BlockHash = Data
+
+struct CryptographicParameters {
+    let onChainCommitmentKey: String
+    let bulletproofGenerators: String
+    let genesisString: String
+}
+
+class Client {
```

Comment:
We should probably called this GRPCClient in order to be consistent with the other implementations

Reply:
In Rust SDK it's called [`Client`](https://github.com/Concordium/concordium-rust-sdk/blob/main/src/v2/mod.rs#L97), in NodeJS SDK it's [`ConcordiumGRPCClient`](https://github.com/Concordium/concordium-node-sdk-js/blob/main/packages/sdk/src/grpc/GRPCClient.ts#L64) so it looks to me like there's no consistent naming to follow. If you like that name better I'm fine with that (would prefer `GrpcClient` casing though ;)

Reply:
The reason it's called just "Client" in Rust is that it is not really important for users that it uses GRPC.

It also follows naming of other "Client" in Rust libraries, such as the one in reqwest (which is not called HTTPClient either).

Reply:
Makes sense - let's stick with 'Client' then